inst/extdata/metadata-silverman-ag.csv

```diff
@@ -1,6 +1,6 @@
 "BiocVersion","SourceVersion","Coordinate_1_based","Species","TaxonomyId","SourceVersion.1","Genome","SourceUrl","DataProvider","Maintainer","Title","Description","SourceType","RDataClass","DispatchClass","RDataPath","Tags"
-"3.13",NA,TRUE,"Homo sapiens","9606",2021-01-06 19:45:53,NA,"https://doi.org/10.1186/s40168-018-0584-3",NA,"Sudarshan Shetty <sudarshanshetty9@gmail.com>","Silverman Artificial Gut microbiome counts","Count matrix for the Silverman microbiome dataset","CSV","matrix","Rds","microbiomeDataSets/silverman-ag/counts.rds","Microbiome"
-"3.13",NA,TRUE,"Homo sapiens","9606",2021-01-06 19:45:53,NA,"https://doi.org/10.1186/s40168-018-0584-3",NA,"Sudarshan Shetty <sudarshanshetty9@gmail.com>","Silverman Artificial Gut row data","Row data for the Silverman microbiome dataset","CSV","DFrame","Rds","microbiomeDataSets/silverman-ag/rowdata.rds","Microbiome"
-"3.13",NA,TRUE,"Homo sapiens","9606",2021-01-06 19:45:53,NA,"https://doi.org/10.1186/s40168-018-0584-3",NA,"Sudarshan Shetty <sudarshanshetty9@gmail.com>","Silverman Artificial Gut sample data","Sample data for the Silverman dataset","CSV","DFrame","Rds","microbiomeDataSets/silverman-ag/coldata.rds","Microbiome"
-"3.13",NA,TRUE,"Homo sapiens","9606",2021-01-06 19:45:53,NA,"https://doi.org/10.1186/s40168-018-0584-3",NA,"Sudarshan Shetty <sudarshanshetty9@gmail.com>","Silverman Artificial Gut tree data","Feature tree data for the Silverman dataset","TXT",NA,"FilePath","microbiomeDataSets/silverman-ag/rowtree.tre.gz","Microbiome"
-"3.13",NA,TRUE,"Homo sapiens","9606",2021-01-06 19:45:53,NA,"https://doi.org/10.1186/s40168-018-0584-3",NA,"Sudarshan Shetty <sudarshanshetty9@gmail.com>","Silverman Artificial Gut reference sequence data","Feature reference sequence data for the Silverman dataset","TXT",NA,"FilePath","microbiomeDataSets/silverman-ag/refseq.fasta.gz","Microbiome"
+"3.13",NA,TRUE,"Homo sapiens","9606",2021-01-07 17:38:30,NA,"https://doi.org/10.1186/s40168-018-0584-3",NA,"Sudarshan Shetty <sudarshanshetty9@gmail.com>","Silverman Artificial Gut microbiome counts","Count matrix for the Silverman microbiome dataset","CSV","matrix","Rds","microbiomeDataSets/silverman-ag/counts.rds","Microbiome"
```

Comment:
These changes seam to be unnecessary. Any reason why they are in here?

Reply:
> This PR is for the dataset for which we could not get raw count and decided to not include it. But never decided on deleting this PR.

I know, but the changes to the silverman-ag set should not be part of this PR, should they? They are already on Bioconductors ExperimentHub and I don't see changes related to these data files in the PR.

Reply:
btw: Congrats on getting this done 😄 I know it took some time. Thanks for patience and perseverence

Reply:
Yes the Silverman files should not have been modified in this PR. I may have accidentally done something. I think we should consider this as a dead PR. I don't think we see iPOP anytime soon here...

Reply:
As I said: You can merge from master and solve the problem there. No harm done

Reply:
Okay will do it in coming days :)